--- Python-3.2.4/Lib/test/test_sys.py.org	2013-04-06 02:38:39.000000000 -0500
+++ Python-3.2.4/Lib/test/test_sys.py	2013-05-03 15:23:12.709474248 -0500
@@ -786,6 +786,8 @@
         # (PyTypeObject + PyNumberMethods + PyMappingMethods +
         #  PySequenceMethods + PyBufferProcs)
         s = vsize('P2P15Pl4PP9PP11PI') + struct.calcsize('16Pi17P 3P 10P 2P 2P')
+        if hasattr(sys, 'getcounts'):
+            s += 20
         check(int, s)
         # class
         class newstyleclass(object): pass
