--- Lib/test/test_gdb.py.org	2013-05-06 15:21:00.620035609 -0500
+++ Lib/test/test_gdb.py	2013-05-06 15:25:23.665044222 -0500
@@ -110,6 +110,14 @@
         # Generate a list of commands in gdb's language:
         commands = ['set breakpoint pending yes',
                     'break %s' % breakpoint,
+
+        # GDB as of Fedora 17 onwards can distinguish between the
+        # value of a variable at entry vs current value:
+        #   http://sourceware.org/gdb/onlinedocs/gdb/Variables.html
+        # which leads to the selftests failing with errors like this:
+        #   AssertionError: 'v@entry=()' != '()'
+        # Disable this:
+        'set print entry-values no',
                     'run']
         if cmds_after_breakpoint:
             commands += cmds_after_breakpoint
@@ -159,8 +167,16 @@
                           'Do you need "set solib-search-path" or '
                           '"set sysroot"?\n',
                           '')
+        err = '\n'.join([line
+                          for line in err.splitlines()
+                          if not line.startswith('warning: Unable to open')
+                          if not line.startswith('Missing separate debuginfo for')
+                          if not line.startswith('Try: yum --disablerepo=')
+                          # In case 'set print entry-values no' failed:
+                          if not line.startswith('Undefined set print command')])
 
         # Ensure no unexpected error messages:
+        self.maxDiff = None
         self.assertEqual(err, '')
 
         return out
